src/components/form/constants.ts

```diff
@@ -0,0 +1,3 @@
+export const GROUP_PLACEHOLDER = 'Enter the name of an existing group...';
```

Comment:
Moving constants into their own files to get them out of the way

Reply:
Not wild about this, prefer it closer to where it is used, but since we will want to consider localizing this project, imagine that ultimately will dictate where these live.